woocommerce.php

```diff
@@ -19,47 +19,26 @@
 }
 
 /**
- * Load all the packages.
+ * Load core packages and the autoloader.
  *
- * We want to fail gracefully if `composer install` has not been executed yet, so we are checking for the autoloader.
- * If the autoloader is not present, let's log the failure and display a nice admin notice.
+ * The new packages and autoloader require PHP 5.6+. If this dependency is not met, do not include them. Users will be warned
```

Comment:
So does this make a breaking change in 3.7 for users on < 5.6? Especially if they use the REST API or blocks currently?

Reply:
@timmyc yes - I brought this up in Proton today. They are happy with this min-requirement, we just need to look at how core enforces this outside of this PR. If core does it, the version check code in this branch can go.